retrofit2/src/main/java/com/linecorp/armeria/client/retrofit2/RetrofitMeterIdPrefixFunctionBuilder.java

```diff
@@ -51,10 +53,24 @@ public RetrofitMeterIdPrefixFunctionBuilder withServiceTag(String serviceTagName
         return this;
     }
 
+    /**
+     * Adds a Retrofit interface service class that will allow to record actual path for each method in
+     * addition to method name.
+     *
+     * @param serviceClass the class defining client with Retrofit.
+     */
+    public RetrofitMeterIdPrefixFunctionBuilder withServiceClass(Class<?> serviceClass) {
```

Comment:
What do you think about making this method package-private and adding `RetrofitMeterIdPrefixFunction.builder(String, Class)` and `of(String, Class)` which sets the service class at construction time?

Reply:
If I understood your idea correctly we don't need this method at all if will provide new package private constructor.

Reply:
Ah, I understood the way I changed will not work because class package private.
At the same time it feels strange to me to add such builder methods to the `RetrofitMeteridPrefixFunction` class because as a result we would return another specialization.

I feel that it would be better to make `RetrofitClassAwareMeterIdPrefixFunction` class either public or provide method to build this class in the builder (actually revert my last changes and add one more method to builder).

What do you think? Will you prefer way that you suggested initially or add more methods to builder?